ninja-servlet/src/main/java/ninja/servlet/ContextImpl.java

```diff
@@ -134,6 +143,56 @@ public void setRoute(Route route) {
     }
 
     @Override
+    public void parseMultipart() {
+
+        FileItemIterator it = getFileItemIterator();
+        if (it == null) {
+            return;
+        }
+        try {
+            while (it.hasNext()) {
+                FileItemStream item = it.next();
```

Comment:
we usually use very loooong variable names. FileItemStream item should become FileItemStream fileStreamItem... FileItemIterator it = getFileItemIterator() should become FileItemIterator fileItemIterator = getFileItemIterator() and so on... tiny thing, but helps to keep the codebase readable and reasonable similar :)